images/pulumi/configs/latest.apko.yaml

```diff
@@ -15,9 +15,9 @@ contents:
     - aspnet-7-targeting-pack
     # for Pulumi Python support
     - pulumi-language-python
-    - python3
-    - python3-dev
-    - py3-pip
+    - python-3.11
```

Comment:
add a comment to explain why its pinned to 3.11

Reply:
Done